frontend/src/pages/device/Snapshots/index.vue

```diff
@@ -265,7 +255,25 @@ export default {
         onSnapshotCancel () {
             this.busyMakingSnapshot = false
         },
-
+        async downloadSnapshotPackage (snapshot) {
+            const ss = await SnapshotApi.getFullSnapshot(snapshot.id)
+            const owner = snapshot.device || snapshot.project
```

Comment:
This data manipulation/transformation seems to be similar to one done before in the applucation/Snapshots.vue. If moving the transformer in the BE is not a viable option, using a common snapsot transformer/mapper would be the way to do it

Reply:
resolved in [15001ab](https://github.com/FlowFuse/flowfuse/pull/3885/commits/15001ab22c59d4581f2116b97633c5c418756099)